packages/Algernon.package/ALGActionVerbFilter.class/class/helpTexts.st

```diff
@@ -7,7 +7,6 @@ helpTexts
 		add: 'browse'->'Open a given class in the Browser.';
 		add: 'fileout'->'Generates a .st file in Resources of squeak image Folder';
 		add: 'open'->'Open an app, tool or morph';
-		add: 'saverepo'->'Saves to Repository with a working copy';
```

Comment:
I suppose this keyword is already outdated or is there another reason it was removed here?

Reply:
Maybe we could leave 'save' as example for a menu item?